pkg/internal/venafi/sign.go

```diff
@@ -152,6 +152,12 @@ func (v *Venafi) Sign(csrPEM []byte, duration time.Duration, customFields []inte
 
 func newVRequest(cert *x509.Certificate) *certificate.Request {
 	req := certificate.NewRequest(cert)
+
+	if len(cert.Subject.Organization) == 0 {
```

Comment:
Hm, how does Venafi like it if we specify `cert.Subject.Organization[0] = ""` (i.e. a slice with length '1' but the first element has length '0')?

Reply:
Can we use CRD validation to disallow explicit empty subject fields? i.e. disallow people specifying an empty string as an organization, postCode etc?

Reply:
Ah wait, `Organization` is already a `string` and not a `[]string` like many other subject fields 🤦

Reply:
Wait, doubletake again.. it _is_ a `[]string`.. My comment applies! 😄

Reply:
Haven’t tested an empty string but it seems to be reading the org as a string so not too sure if that will work

Reply:
I can confirm that it does need a non empty string

Reply:
I think we may be able to add a restriction in our API to ensure that list entries are non-empty to avoid this being a big issue for users (perhaps a `minLength` parameter?)

Reply:
What do you mean exactly? A minLength on `organnizations`?